content/security/committers.md

```diff
@@ -42,9 +42,9 @@ be subscribed to the list.
 
 <h2 id="possible">Handling a possible vulnerability<a class="headerlink" href="#possible" title="Permanent link">&para;</a></h2>
 
-Here is a typical process for handling a possible security vulnerability.
```

Comment:
replace "a typical" with "the default".   We don't want projects stalled thinking they need to develop a process if they are using the standard one (which nearly all are).

Reply:
I think this is good - this is the text being removed in this PR rather than the text being added. i.e. my change is along the same lines as your comment.